### ISSUE TITLE
set_pose makes mesh black 

### ISSUE THREAD

**Issue by ZhengyiLuo** [mmatl/pyrender#107]
When using set_pose to set pose to certain nodes, the rendered mesh become black as shown in the images.  It is not consistent somehow: over some poses the mesh would be black and some won't. 
![image](https://user-images.githubusercontent.com/13114740/77838519-bb289280-7142-11ea-96cf-36792ca0af6b.png)

**Comment by jiunyen-ching** [comment]
In the black image, it seems like the wheels are barely visible. Could it be due to the lighting node settings?

**Comment by ZhengyiLuo** [comment]
> In the black image, it seems like the wheels are barely visible. Could it be due to the lighting node settings?

No it does not happen when I do not use the set_pose on pyrender and directly apply the transformation on the mesh.

**Comment by jiunyen-ching** [comment]
Upon further inspection, I noticed that the child node/mesh will appear black when the determinant of the matrix = -1. Vice versa, it will have color when its matrix's determinant = 1. 

The bottle in the middle is for reference while the 'moving' bottle has its pose set with ortho_group.rvs(3).

![compile](https://user-images.githubusercontent.com/25114361/79597095-45499280-8114-11ea-8fe5-5a4b510c693b.png)

Despite the bug with the color, the depth map is rendered correctly.
![depth_bottle](https://user-images.githubusercontent.com/25114361/79630293-b031b280-8182-11ea-97a8-2558195183a6.png)

Also tested out with 2 child nodes to verify consistency of the bug; meshes of child nodes appears black when determinant of its' pose matrix = -1. The pose of the parent node has a determinant = -1 but does not exhibit the behavior of the bug.

![bottle_bottle_drill](https://user-images.githubusercontent.com/25114361/79630137-66949800-8181-11ea-8dfd-6a61ec44a42f.png)

**Comment by ZhengyiLuo** [comment]
Indeed! I noticed that I have been sampling from ortho_group instead of special_ortho_group from scipy. 

I guess not handling reflection well is okay?

**Comment by jiunyen-ching** [comment]
I see. I am not familiar with how the determinant of the pose affects reflection.
